legacy-example-raw-http-run.js

```diff
@@ -0,0 +1,10 @@
+
+
+var Falafel = require('./');
```

Comment:
Whats the purpose of this file?

(the whole file, not the require)

Reply:
It mimics the `main.js` file - so this one runs the `exampleRawHttpRequest` folder. None of these or the example files are part of the production build.